build/containers/archive-analysis/README.md

```diff
@@ -15,6 +15,8 @@ $ podman run \
 
 This command starts the container, which runs Redis and Grafana (inside the container) and loads all PCP archives of the selected directory on the host into Redis. You can point your browser to `http://localhost:3000/d/pcp-archive-analysis/pcp-archive-analysis` and can start inspecting the archives using Grafana. 
 
+Optionally add `-e STREAM_MAXLEN_ADJUST=<value>` (default is 86400) to `podman run` to adjust the _stream.maxlen_ parameter in order to increase the number of loaded timeseries entries. Useful when loading archives covering more than one day.
+
```

Comment:
I'd prefer if we could make this a more general solution, rather than for just one variable.

I suggest we name this variable PMSERIES_STREAM_MAXLEN and also allow PMSERIES_STREAM_EXPIRE to be set.

Reply:
Well, a problem is that the container is somewhat meant to be immutable. As you can see it's not too convenient to even simply export a dynamic modification of a file. So not sure how much generic you'd wanna go :D , perhaps it would be easier to supply a custom pmproxy.conf file as a whole instead then - but a question arises when it's easier to simply bash into the container and modify & service-restart it manually inside the container.

We can rename, sure, although I'm not sure if we explicitly need to adjust the "expire" config. IIUC that's pretty long by default anyway.

TBH I think it's a question about what usecases is this container for - I personally use it only for quick one-off inspection of pcp archives, for which I only encounter the limitation for number of entries (stream.maxlen) so far. I think that's it's primary meant use case and I'm not sure we want to encourage any other use. (i.e. if a user needs anything more than this container offers, then I'd say a better way is to set up the environment in OS/VM and optionally use pmseries directly)